sematic/db/models/factories.py

```diff
@@ -35,6 +37,9 @@ def make_run_from_future(future: AbstractFuture) -> Run:
         parent_id=(
             future.parent_future.id if future.parent_future is not None else None
         ),
+        nested_future_id=(
```

Comment:
Yes but it never really mattered until now. Because now futures can have nested_futures from the get-go without having gone through the resolution loop.